scripts/local_env_create_vm

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+## Shell Opts ----------------------------------------------------------------
+
+set -o pipefail
+set -xeuo
+
+
+print_success_message () {
+    CONTENT="
+$(tput bold)Local environment setup success! VM IP: $1. Use 'ssh zuul@$1' to connect
+
+Don't know how to continue? To deploy EDPM:
+
+ssh zuul@$1
+cd /home/zuul/src/ci-framework
+ansible-playbook -i localhost -c local -e @scenarios/centos-9/local-env.yml deploy-edpm.yml
+
+$(tput sgr0)"
+
+    COLS="$(echo "$CONTENT" | wc -L)"
+    printf '%*s' "${COLS}" '' | tr ' ' \#
+    printf "$CONTENT"
+    printf '%*s\n' "${COLS}" '' | tr ' ' \#
+}
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Local env needs sudo permissions"
+    exit 1
+fi
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+CIFMW_LOCAL_ENV_BASE_DIR="${1}"
+CIFMW_LOCAL_ENV_BACKING_IMG_URL="${2}"
+CIFMW_LOCAL_ENV_PULL_SECRET_PATH="${3}"
+CIFMW_LOCAL_ENV_CALLING_USER=$([ -z "$SUDO_USER" ] && echo "root" || echo "$SUDO_USER")
+CIFMW_LOCAL_ENV_CALLING_USER_HOME=$(eval echo "~$CIFMW_LOCAL_ENV_CALLING_USER")
+
+CIFMW_LOCAL_ENV_IMAGES_DIR="${1}/images"
```

Comment:
since ${1} is already assigned to a "human readable var name", we should use it. Note that it should be user writable - the default `/opt/ci-framework` isn't writable by unprivileged user.